pkg/printers/internalversion/describe.go

```diff
@@ -2987,13 +2987,64 @@ func describeNetworkPolicy(networkPolicy *networking.NetworkPolicy) (string, err
 		w := NewPrefixWriter(out)
 		w.Write(LEVEL_0, "Name:\t%s\n", networkPolicy.Name)
 		w.Write(LEVEL_0, "Namespace:\t%s\n", networkPolicy.Namespace)
+		w.Write(LEVEL_0, "Created on:\t%s\n", networkPolicy.CreationTimestamp)
 		printLabelsMultiline(w, "Labels", networkPolicy.Labels)
 		printAnnotationsMultiline(w, "Annotations", networkPolicy.Annotations)
-
+		describeNetworkPolicySpec(networkPolicy.Spec, w)
 		return nil
 	})
 }
 
+func describeNetworkPolicySpec(nps networking.NetworkPolicySpec, w PrefixWriter) {
+	w.Write(LEVEL_0, "Spec:\n")
+	w.Write(LEVEL_1, "Pod Selector: ")
+	if (nps.PodSelector.MatchLabels == nil || len(nps.PodSelector.MatchLabels) == 0) &&
+		(nps.PodSelector.MatchExpressions == nil || len(nps.PodSelector.MatchExpressions) == 0) {
+		w.Write(LEVEL_2, "<none> (Allowing all traffic to all pods in this namespace)\n")
```

Comment:
"all traffic" makes it sound like this has `ingress: [{}]`. Maybe something like "Allowing the specified traffic to all pods in this namespace"